### ISSUE TITLE
Plugin export not Working (Can't download artifact osgi.bundle,com.sun.jna)

### ISSUE THREAD

**Issue by MarcelHaldimann** [eclipse-platform/eclipse.platform#1484]
Let's make sure issue is not already fixed in latest builds first.

- [x] I verified I can reproduce this issue against [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/)

## Issue

Do you have any idea what we are doing wrong? Or can you point us in the right direction? 

We assume that the dependency `com.sun.jna 5.14.0` is simply no longer found in the p2 repo. In the Maven repo it has been moved to `net.java.dev.jna`. 

![Screenshot 2024-07-26 at 08 28 33](https://github.com/user-attachments/assets/ae431d82-8044-4e3f-abf9-476cf014c6b0)

We currently have a problem exporting a product from Eclipse. We have an application which we update to the new version approximately every year. We wanted to do this this week. We are currently on 2023-03 and everything is still working as usual. 

When exporting the product, we receive the following error message: 

![Screenshot 2024-07-26 at 08 01 44](https://github.com/user-attachments/assets/3db91602-4eea-45e6-8c52-d587a32163a8)

We have tested it with 2024-06, 2024-03, also under Windows, MacOS, AppleSilicon or x86 with Rosetta2. No difference. 

After a few hours we decided to try it with the generated demo project from Eclipse. The same problem here too. 

## Steps to reproduce

1. Open Eclipse
2. Create new Plug-in Project
4. Settings like the following
![Screenshot 2024-07-26 at 08 10 51](https://github.com/user-attachments/assets/bf3685f4-6af2-4daa-8ca1-7bedd236a9e5)

![Screenshot 2024-07-26 at 08 10 59](https://github.com/user-attachments/assets/913f4cb9-a12d-4f7e-83c0-4f85a4ce362e)

![Screenshot 2024-07-26 at 08 11 13](https://github.com/user-attachments/assets/01ee6e29-13b4-41a1-85b3-1bc0e08811e4)

(I can start it from Eclipse without any problems)
![Screenshot 2024-07-26 at 08 14 00](https://github.com/user-attachments/assets/3564f858-6282-4f74-bc0c-98c59b8d02b6)

[eclipse-rcp-jna-problem.zip](https://github.com/user-attachments/files/16387378/eclipse-rcp-jna-problem.zip)

5. Try to export the product (plugin based)

![Screenshot 2024-07-26 at 08 24 49](https://github.com/user-attachments/assets/7b5bd806-b6cb-40e2-9eac-369439e1c5f2)

![Screenshot 2024-07-26 at 08 16 29](https://github.com/user-attachments/assets/b73a61fc-5f8f-47c2-b362-bf17738f9012)

## Here is some relevant log output

```
!ENTRY org.eclipse.equinox.p2.director 0 0
!MESSAGE Overall install request is satisfiable
!SUBENTRY org.eclipse.equinox.p2.director 0 0
!MESSAGE Add request for ch.ionesoft.ui.test 1.0.0.202407251518 (test 1.0.0.202407251518) is satisfiable
!ENTRY org.eclipse.equinox.p2.engine 4 4
!MESSAGE An error occurred while collecting items to be installed
!SUBENTRY org.eclipse.equinox.p2.engine 4 0
!MESSAGE session context was:(profile=profile, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
!SUBENTRY org.eclipse.equinox.p2.artifact.repository 4 0
!MESSAGE Problems downloading artifact: osgi.bundle,com.sun.jna,5.14.0.v20231211-1200.
```

or

```
!MESSAGE Can't download artifact osgi.bundle,com.sun.jna,5.14.0.v20231211-1200 required by 
```

## Tested under this environment:

MacOS x86 -> 2024-06
Windows x86 -> 2024-06
MacOS aarch64 -> 2024-03

## Community

- [x] I understand reporting an issue to this OSS project does not mandate anyone to fix it. Other contributors may consider the issue, or not, at their own convenience. The most efficient way to get it fixed is that I fix it myself and contribute it back as a good quality patch to the project.

**Comment by laeubi** [comment]
See 
- https://github.com/eclipse-pde/eclipse.pde/issues/862

**Comment by MarcelHaldimann** [comment]
Hello Christoph 

Yes, we tried to switch the process to Tycho about 6-7 months ago. We failed successfully ;). Now we have tried it again these days and everything has now worked. We were able to solve all the problems we had. 2-3 things are still not clear why we had to change it. But now we could successfully export a product. 

I am closing this issue. 

Thanks for your feedback and the work.
Marcel
